sql/plan/create_index.go

```diff
@@ -49,6 +50,7 @@ func NewCreateIndex(
 		Exprs:  exprs,
 		Driver: driver,
 		Config: config,
+		Async:  config["async"] != "false",
```

Comment:
Is it ok using the configuration keys directly as strings instead of parameterize them through constants?

Reply:
I don't think async is gonna be called anything else anytime soon, and it's not used anywhere else, so no need to add two more constants for this